### DISCUSSION TITLE
Docs are missing

### DISCUSSION THREAD

**Discussion by vlados** [shopperlabs/framework#93] [General] [answered]
It seems like a great product but the missing of extended documentation is making the use of the project hard.
For me I need to understand the concept of a package and how everything is connected so I can understand how to build a project on top of that package
Is there any way the community can help with this?

**Comment by mckenziearts** [comment]
For the moment the documentation is not public put a the end of this month i will publish the repo.

**Comment by mckenziearts** [comment, marked as answer]
Documentation is available https://laravelshopper.dev
